tests/inline.js

```diff
@@ -168,15 +170,33 @@ describe( 'InlineEditor', () => {
 			return InlineEditor.create( editorElement, { plugins: [ Paragraph ] } )
 				.then( newEditor => {
 					editor = newEditor;
+
+					const schema = editor.document.schema;
+
+					schema.registerItem( 'bold' );
+					schema.allow( { name: 'bold', inside: '$root' } );
+					schema.allow( { name: '$text', inside: 'bold' } );
+
+					buildViewConverter().for( editor.data.viewToModel )
+						.fromElement( 'b' )
+						.toElement( 'bold' );
+
+					buildModelConverter().for( editor.data.modelToView )
+						.fromElement( 'bold' )
+						.toElement( 'b' );
+
+					buildModelConverter().for( editor.editing.modelToView )
```

Comment:
> Basic styles and paragraph features are dev deps of this package anyway.

With these features, this test would pass without the fix. This is actually why the old test passed and we didn't notice it. View representation of those features is the same as data. We discovered this bug thanks to captioned image widget because it's different in editing and editor data.

These converters I created basically simulate a feature which looks different in editing and in data. If the fix fails and the engine overrides the data output of the editor, the user will get the editing view output instead of the actual editor data. This tests checks that thing.